nodes/viz/texture_viewer.py

```diff
@@ -120,6 +128,13 @@ class SvTextureViewerNode(bpy.types.Node, SverchCustomTreeNode):
         update=updateNode
     )
 
+    bitmap_save = EnumProperty(
+        items=bitmap_save_list,
+        description="offers bitmap saving",
+        default="PNG",
+        update=updateNode
```

Comment:
if you want to trigger a call to `process` you add `update=updateNode`, but in this case you just want to `set` the property for the save operator to us.  Drop `update=updateNode` ,  will be ok.